src/basic/vx_config/config.tab.cc

```diff
@@ -1389,7 +1389,7 @@ yyparse (void)
 
   YYDPRINTF ((stderr, "Starting parse\n"));
 
-  yystate = 8;
+  yystate = 0;
```

Comment:
```suggestion
  yystate = 8;
```
This needs to remain a value of 8. That enables empty config files to be parsed without error. However, this is automatically generated code. So every time it gets regenerated, it resets that 8 to 0. I suppose we need to update the Makefile.am logic to switch the 0 back to 8 after regenerating it. Ugh.

Reply:
@JohnHalleyGotway I'm not sure why that change didn't get picked up in the pull from develop... In any case, I will make that change and commit it.  Sorry about that!